demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,70 @@
+<h5>Navigate through options in the popup</h5>
+
+<table class="table mt-4">
+  <tbody>
+    <tr>
+      <td><kbd>ArrowUp</kbd></td>
+      <td>Highlight above option</td>
+    </tr>
+    <tr>
+      <td><kbd>ArrowDown</kbd></td>
+      <td>Highlight below option</td>
+    </tr>
+    <tr>
+      <td><code>mouseenter</code></td>
+      <td>Highlight option under the mouse pointer</td>
+    </tr>
+  </tbody>
+</table>
+
+<p>
+  Note that starting keyboard navigation will always start from the currently activated option, no matter if activated from the keyboard or the mouse.
+</p>
+
+
+
+<h5>Select an option</h5>
+
+<p>Selecting means committing to the data model. It will display the currently selected option in the input. It will also make the popup close.</p>
+
+<table class="table mt-4">
+  <tbody>
+    <tr>
+      <td><kbd>Enter</kbd> ● <kbd>Tab</kbd></td>
+      <td>Select active option</td>
+    </tr>
+    <tr>
+      <td><code>click</code> on option in the popup</td>
+      <td>Select option under mouse pointer</td>
+    </tr>
+  </tbody>
+</table>
+
+
+
+<h5>Dismiss the popup</h5>
+
+<p>
+  Dismissing the popup consists in closing the popup without selecting an option.
```

Comment:
Stating the obvious

Reply:
Well, I guess I didn't pay attention enough to the fact that the word "dismiss" is self-explanatory in this case, I guess.